src/generate/generate_from_csv.py

```diff
@@ -296,7 +297,7 @@ def generate_from_csv(args):
         "scenario": {
             "start_time": start.isoformat(),
             "interval": interval.total_seconds() // 60,
-            "n_intervals": (stop - start) // interval,
+            "stop_time": stop.isoformat(),
```

Comment:
ok, if this does not lead to problems with other scripts?
However this should then be the same for all generation scripts. In from_simbev and from_statistics it is still this:
```
"scenario": {
            "start_time": start.isoformat(),
            "interval": interval.total_seconds() // 60,
            "n_intervals": (stop - start) // interval,
            "discharge_limit": args.discharge_limit,
        },
````

Reply:
updated in generate_from_statistics to use stop_time as well. Both options are valid. Decided on a per-script-basis what option is simpler.